test/tests.js

```diff
@@ -612,6 +611,16 @@ describe( "MessageFormat", function () {
         expect(mfunc({"VAR":"big"})).to.eql("This is BIG.");
       });
 
+      it("should use bidiStructuredText MessageFormat formatter", function () {
+        var mf = new MessageFormat('en', null, {"bidiStructuredText": MessageFormat.formatters.bidiStructuredText} );
+        var mfunc = mf.compile("{0} >> {1}");		
+        expect(mfunc([ "first_english_word", "SECOND_ARABIC_WORD" ])).to.equal('\u200efirst_english_word\u200e >> \u200eSECOND_ARABIC_WORD\u200e');
+
+        mf = new MessageFormat('ar-EG', null, {"bidiStructuredText": MessageFormat.formatters.bidiStructuredText} );
+        var mfunc = mf.compile("{0} >> {1}");		
+        expect(mfunc([ "first_english_word", "SECOND_ARABIC_WORD" ])).to.equal('\u200ffirst_english_word\u200f >> \u200fSECOND_ARABIC_WORD\u200f');
+      });
+
```

Comment:
As we're dealing with Unicode, there's no reason why `SECOND_ARABIC_WORD` isn't here a word in actual Arabic.